docs/user-guide.md

```diff
@@ -202,6 +202,10 @@ Liberty Tools editing assistance provides code completion in configuration and a
 
 ![Eclipse LSP4MP microprofile-config.properties completion](images/LSP4MP-mp-properties-completion.png)
 
+3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a supported Jakarta EE or MicroProfile diagnostic. A pop-up dialog containing the diagnostic message and a quick-fix (code action) appears.
```

Comment:
```suggestion
3. To use Jakarta EE-specific and MicroProfile-specific quick-fixes, hover over a Jakarta EE or MicroProfile diagnostic. A pop-up dialog containing the diagnostic message and a quick-fix suggestion will appear. To see more quick-fix suggestions, click on the "More actions..." link.
```

Reply:
same here:

click on the **More actions...** link.

Reply:
👍

Reply:
Done.